uli-community/lib/uli_community/media_processing/vid_vec_rep_clip.ex

```diff
@@ -0,0 +1,19 @@
+defmodule UliCommunity.MediaProcessing.VidVecRepClip do
+  use Export.Python
+
+  @python_path Path.expand("lib/python")
+
+  def get_embedding(file_path) do
+    {:ok, py} = Python.start(python_path: @python_path)
```

Comment:
this might be something you do in an init function.
my sense is that you can create a port that talks to the python process and keep it open. This py has to be a reference to that port. so best to create it once and then use throughout the functions that talk to the port.